packages/blade/src/utils/getPlatform/getPlatform.ts

```diff
@@ -0,0 +1,19 @@
+export type PlatformTypes = 'browser' | 'node' | 'react-native' | undefined;
+
+const getPlatformType = (): PlatformTypes => {
+  if (typeof navigator !== 'undefined' && navigator.product === 'ReactNative') {
+    return 'react-native';
+  }
+
+  if (typeof document !== 'undefined') {
+    return 'browser';
+  }
+
+  if (typeof process !== 'undefined') {
+    return 'node';
+  }
+
+  return undefined;
```

Comment:
How about throwing an error here? If it's none of the above platforms, could be some bug actually?

Reply:
could be any other platform too in the future like watchOS or something. regardless i need to return since it's a function expecting a type, else TS will beat me off :D 

I'll add `console.error` though.

Reply:
Would probably be better if we return consistent string here? 

Like `return "unknown"`

That we atleast we don't have to check for undefined when using this function. We can just assert on string itself.